FPM.ftd

```diff
@@ -24,3 +24,10 @@ zip: github.com/fifthtry/bling/archive/refs/heads/main.zip
 - Message: /chat/
 - Tip: /tip/
 - Quote: /quote/
+  - one: /dev/intro/
```

Comment:
@priyanka9634 change - Quote with - Quotes! and add all other quotes as it's children! Names should be - Quote-1, - Quote-2, - Quote-3 and so on... for now!